src/analysis/normed_space/basic.lean

```diff
@@ -583,7 +583,7 @@ end
 continuous. -/
 @[priority 100] -- see Note [lower instance priority]
 instance normed_uniform_group : uniform_add_group α :=
-⟨(lipschitz_with.prod_fst.sub lipschitz_with.prod_snd).uniform_continuous⟩
+⟨((@lipschitz_with.prod_fst α α _ _).sub lipschitz_with.prod_snd).uniform_continuous⟩
```

Comment:
This is interesting: the `@` is not needed if `α` is a `semi_normed_group`, see #6746 (the PR I split this one from). In practice this means that I can go back to the original version in a future PR.